pal_talos/assets/arm/arm_11.stl

```diff
@@ -0,0 +1 @@
+arm_1.STL
```

Comment:
We had to do this due to the following issue: https://github.com/deepmind/mujoco/issues/163. We usually reuse the meshes instead of duplicating them, that's the reason for going with the softlinks.

Reply:
I think this is no longer an issue after [this commit](https://github.com/google-deepmind/mujoco/commit/d12fe594d98ee0500fa6bb7513f3faf3d9f7488a), can you please check is this is still an issue?

We are in the process of merging and would love to avoid all these duplicated files to avoid bloat.

Honestly even if the issue is not resolved I prefer to resolve it first. MuJoCo should not be forcing users to needlessly duplicate files 🙂

Reply:
Hello @yuvaltassa,

These are just simlinks and are not a duplication, so it shouldn't be a problem.

Sure, I can check this out and get back to you